src/main/webapp/profile.jsp

```diff
@@ -0,0 +1,65 @@
+<%--
```

Comment:
It looks like this file got added twice.  It seems like the other one (webapp/WEB-INF/view/profile.jsp) is the right location.